crm_claim_rma/models/account_invoice_line.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright 2015 Vauxoo
+#    Copyright 2013 Camptocamp
+#    Copyright 2009-2013 Akretion,
+#    Author: Emmanuel Samyn, Raphaël Valyi, Sébastien Beau,
+#            Benoît Guillot, Joel Grand-Guillaume,
+#            Osval Reyes, Yanina Aular
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import api, models
+
+
+class AccountInvoiceLine(models.Model):
+
+    """
+    Account Invoice Line
+    """
```

Comment:
Unnecesary doc, just doc classes if you will put information with meaning, if you will only duplicate the class name itself, it is better to not put any doc here.